samples/AntiforgerySample/FormPostSampleMiddleware.cs

```diff
@@ -40,7 +40,7 @@ public class FormPostSampleMiddleware
 </html>";
 
                 var tokenSet = _antiforgery.GetAndStoreTokens(context);
-                await context.Response.WriteAsync(string.Format(page, _options.FormFieldName, tokenSet.FormToken));
+                await context.Response.WriteAsync(string.Format(page, _options.FormFieldName, tokenSet.RequestToken));
             }
             else if (context.Request.Method == "POST")
```

Comment:
Probably having a simple ajax scenario as part of this sample would be helpful to users. You can file a bug and we can add it later.